IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/README_JP.md

```diff
@@ -334,4 +334,4 @@ wolfSSL_CTX_use_certificate_buffer あるいはwolfSSL_CTX_use_certificate_chain
 上記制限1 ~ 4は次版以降のTSIPによって改善が見込まれています。
 
 ## 11. サポート
-ご質問・ご要望は、info@wolfssl.jp まで日本語でお知らせください。
\ No newline at end of file
+ご質問・ご要望は、info@wolfssl.jp まで日本語でお知らせください。
```

Comment:
What changed here?

Reply:
Good question. I assumed it is a DOS line ending in a POSIX file, or something similar, but let me check.

Reply:
Strangely [Gnome Meld](https://meldmerge.org/) doesn't show any difference on that line. However, given this is the last line, I guess it's a missing line break at the end of that last line. My editor is POSIX compliant and always adds a line break at end-of-line, including the last line. Let me double-check, though.

Reply:
Indeed:
```
$ env LANG=C diff README_JP.orig README_JP.md
240c240
< 実行すると、下記のようなメッセージがRenesas Debug Virtual Consol に表示されます。
---
> 実行すると、下記のようなメッセージがRenesas Debug Virtual Console に表示されます。
301c301
< Renesas Debug Virtual Consol に下記のように表示されていれば、ボード上の TLS サーバーとクライアント間でTLS通信が行われています。
---
> Renesas Debug Virtual Console に下記のように表示されていれば、ボード上の TLS サーバーとクライアント間でTLS通信が行われています。
337c337
< ご質問・ご要望は、info@wolfssl.jp まで日本語でお知らせください。
\ No newline at end of file
---
> ご質問・ご要望は、info@wolfssl.jp まで日本語でお知らせください。
$ 
````
Would you like me to to remove that last line break, or do you prefer your files to be POSIX-compliant?

Reply:
Please leave it unchanged.

Reply:
All the editors I tried ensure the presence of a line break at the end of every line, including the last line. I ended up removing the last character with:
```
sed '$ s/.$//' IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/README_JP.md
```

Reply:
Nope, sed removed the  `。` in addition to the line break. I need to find something else.

Reply:
This did the trick:
```
truncate -s -1 IDE/Renesas/e2studio/RX72N/EnvisionKit/Simple/README_JP.md
```